components/eam/chem_proc/procfiles/cam/mo_imp_sol_cache.F90

```diff
@@ -153,14 +153,14 @@ subroutine imp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, &
 !           do_diag = lev == 5 .and. isec == 2 .and. lchnk == 290
             do_diag = .false.
             if( do_diag ) then
-	      write(*,*) ' '
-	      write(*,*) 'imp_sol: lchnk,lev,isec,cols = ',lchnk,lev,isec,cols
-	      write(*,*) ' '
-	      write(*,*) 'imp_sol: lrxt'
-	      write(*,'(1p,4g20.10)') lrxt(1,:)
-	      write(*,*) 'imp_sol: lhet'
-	      write(*,'(1p,4g20.10)') lhet(1,:)
-	    end if
+               write(*,*) ' '
```

Comment:
Using `write(*,*)` is why these wind up in e3sm.log.  It should be write(iulog,*).  Don't have to do that now.   Looks like there is a lot of those to clean up in the chemistry.

Reply:
I'm happy to make that change for this file, just let me know if I should do that or hold off.

Reply:
go ahead!